src/extension/options-page/DashboardComponents/ActionButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Button, Typography, CircularProgress } from '@material-ui/core'
+import { ButtonProps } from '@material-ui/core/Button'
+
+interface ButtoneProps extends ButtonProps {
+    width?: number | string
+    loading?: boolean
+    // TODO: figure out type
+    [key: string]: any
+}
+
+export default function ActionButton(props: ButtoneProps) {
+    const { width, loading, children, className, ...p } = props
+    return (
+        <Button
+            disabled={loading}
+            startIcon={loading && <CircularProgress size={24} />}
+            className={'actionButton ' + className}
```

Comment:
Use `className('actionButton', className)`.

Reply:
I think this is ok as className may be a space-separated string

Reply:
I think it's okay in this simple case